### PR TITLE
Docs: Update nav titles, add a Reference node, move reference topics

### PR DESCRIPTION
**Pull request by randall-Mysten**

### CODE REVIEW

File: doc/src/siteconfig.json [hunk 1]
```diff
@@ -15,69 +15,7 @@
...
-          },
-          {
-            "title": "White Paper",
-            "link": "https://github.com/MystenLabs/sui/blob/main/doc/paper/sui.pdf",
-            "external": true
-          }
-        ]
-      }
-    ]
+    }
```
Comment (Jibz1) [inline doc/src/siteconfig.json:18]: This is taking out the footer menu 
Screen Shot 2022-10-19 at 7 27 08 PM

Comment (randall-Mysten) [reply doc/src/siteconfig.json:18]: Correct

**Comment by randall-Mysten** [comment]
@Jibz1 Anything else I need to change here before we can merge this?

**Comment by Jibz1** [comment]
reviewing

**Comment by Jibz1** [comment]
@randall-Mysten the footer menu is still missing

**Review by Jibz1 — APPROVED** [review]

**Pull request merged**